package.json

```diff
@@ -80,17 +80,15 @@
     "copy-text-to-clipboard": "^2.2.0",
     "ky": "^0.24.0",
     "micro-match": "^1.0.1",
-    "prettier": "^2.2.1",
     "prop-types": "15.7.2",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-router": "5.2.0",
     "react-router-dom": "5.2.0",
     "react-svg": "^11.2.2",
     "react-textarea-autosize": "^8.3.0",
-    "speakingurl": "14.0.1",
     "stacktrace-js": "^2.0.0",
-    "strip-indent": "^3.0.0",
+    "tickety-tick-formatter": "bitcrowd/tickety-tick-formatter",
```

Comment:
💡 I hadn't yet seen that you can now use such shortcuts for GitHub URLs.

— https://docs.npmjs.com/cli/v7/configuring-npm/package-json#github-urls